<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saas_guide_pay</class>
 <widget class="QWidget" name="saas_guide_pay">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="lds_ComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>162</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_terminal_num">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>162</width>
     <height>30</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>351</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>exmaple</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>台</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>481</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20pt;</string>
   </property>
   <property name="text">
    <string>购买</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>510</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#ff546b;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pay">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>162</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20pt;
color:orangered;
background:transparent;</string>
   </property>
   <property name="text">
    <string>10.00</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>购买费用</string>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/image/q5.png</normaloff>:/image/q5.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
   <property name="outer_stylesheet" stdset="0">
    <string>autoRaise</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>POS机服务</string>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/image/q1.png</normaloff>:/image/q1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
   <property name="outer_stylesheet" stdset="0">
    <string>autoRaise</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>购买期限</string>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/image/q3.png</normaloff>:/image/q3.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
   <property name="outer_stylesheet" stdset="0">
    <string>autoRaise</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>终端数</string>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/image/q4.png</normaloff>:/image/q4.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
   <property name="outer_stylesheet" stdset="0">
    <string>autoRaise</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>350</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
min-height:29px;
}

QRadioButton:indicator{
width:0px;
}

QRadioButton:checked{
background:#68c3de;
border:1px solid lightgray;
border-radius: 2px;
color:white;
}

QRadioButton:unchecked{
background:white;
border:1px solid lightgray;
border-radius: 2px;
color:black;
}</string>
   </property>
   <widget class="QRadioButton" name="radioButton_extend">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>162</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>续期</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_update">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>162</width>
      <height>31</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>升级</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="toolButton_num_key">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/image/widgets_pic/keyboard.png</normaloff>:/image/widgets_pic/keyboard.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>45</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_cloudActivityRuleNUMText">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:orangered;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_cloudActivityRuleDiscountText">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>401</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:orangered;</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_ComboBox</class>
   <extends>QComboBox</extends>
   <header>lds_combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
